docker/docker-compose-cli.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
+
+services:
+  redis:
+    image: redis
+
+  cli:
+    image: ${FAABRIC_CLI_IMAGE}
+    volumes:
+        - ..:/code/faabric
```

Comment:
Just for consistency, I think it would be better to mount at `/usr/local/code/faabric`.

Reply:
I'd actually rather move away from `/usr/local/code` as it's not really a good place to put code (it means only a single user can have their code checked out on a shared machine). Unfortunately there are still a couple of places in Faasm where it assumes `/usr/local/code`. The pattern for our Docker containers should be `/code` for code and `/build` for build.